deps/nghttp2/lib/includes/nghttp2/nghttp2ver.h

```diff
@@ -29,14 +29,14 @@
  * @macro
  * Version number of the nghttp2 library release
  */
-#define NGHTTP2_VERSION "1.32.0"
+#define NGHTTP2_VERSION "1.33.0-DEV"
```

Comment:
Should we be moving to a dev version? Or is this maybe a mistake on their end (based on the linked GitHub release)?

Reply:
I think this is a mistake by me, not having run `configure` in the nghttp2 checkout before copying the files. Thanks for catching!